ReactWindows/ReactNative.Tests/Modules/Clipboard/ClipboardModuleTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
+using ReactNative.Bridge;
+using ReactNative.Modules.Clipboard;
+using System;
+
+namespace ReactNative.Tests.Modules.Clipboard
+{
+    [TestClass]
+    public class ClipboardModuleTests
+    {
+        [TestMethod]
+        public void ClipboardModule_ArgumentChecks()
+        {
+            var module = new ClipboardModule(new ReactContext());
+
+            AssertEx.Throws<ArgumentNullException>(
+                () => module.getString(null),
+                ex => Assert.AreEqual("promise", ex.ParamName));
+        }
+
+        [TestMethod]
+        public void ClipboardModule_GetString_Method()
+        {
+            if (!ReactNative.Bridge.DispatcherHelpers.IsInitialized) return;

```

Comment:
See comment below.

---
In reply to: [55509684](https://github.com/CatalystCode/react-native/pull/233#discussion_r55509684) [](ancestors = 55509684)